src/gui/guiButton.h

```diff
@@ -129,6 +129,20 @@ class GUIButton : public gui::IGUIButton
 		setImage(gui::EGBIS_IMAGE_DOWN, image, pos);
 	}
 
+	//! Sets an image which should be displayed on the button when it is in pressed state.
+	virtual void setHoveredImage(video::ITexture* image=0)

```

Comment:
The code comment is wrong (You probably copied.)

and

```suggestion
	virtual void setHoveredImage(video::ITexture *image = nullptr)
```
.
(See also https://dev.minetest.net/Code_style_guidelines.)

Reply:
You're right, that was a bad copy on my part. I've updated the code accordingly.